common/script/index.coffee

```diff
@@ -822,7 +822,8 @@ api.wrap = (user, main=true) ->
           item = content[type][key]
           price = item.value / 4
         return cb?({code:404,message:":key not found for Content.#{type}"},req) unless item
-        return cb?({code:401, message: i18n.t('notEnoughGems', req.language)}) if (user.balance < price) or !user.balance
+        return cb?({code:403, message: i18n.t('messageNotAvailable', req.language)}) if type is 'hatchingPotions' and not item.canBuy
```

Comment:
It might be better to generalize this so we can flag any buyable item with `canBuy:false` to turn it off, but at the moment we're not consistent in the meaning of `canBuy` across the app. I'll put some work into this next.